phpstan-baseline.neon

```diff
@@ -9795,11 +9795,6 @@ parameters:
 			count: 1
 			path: modules/social_features/social_post/social_post.install
 
-		-
-			message: "#^Call to an undefined method Drupal\\\\Core\\\\Form\\\\FormInterface\\:\\:getEntity\\(\\)\\.$#"
-			count: 1
-			path: modules/social_features/social_post/social_post.module
-
```

Comment:
Finally by replacing hooks we should be able to reduce this monster hidden under the carpet that contains plenty of non-compatible call that should be resolved.